verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -248,7 +251,43 @@ std::vector<verible::lsp::Location> SymbolTableHandler::FindDefinitionLocation(
   const verible::TextStructureView &text = parsedbuffer->parser().Data();
 
   const verible::TokenInfo cursor_token = text.FindTokenAt(cursor);
-  auto symbol = cursor_token.text();
+  return cursor_token.text();
+}
+
+std::optional<verible::lsp::Location>
+SymbolTableHandler::GetLocationFromSymbolTableNode(
+    absl::string_view symbol, const VerilogSourceFile *file_origin) {
+  // TODO (glatosinski) add iterating over multiple definitions
+  verible::lsp::Location location;
```

Comment:
You need `location` much later. I suggest declaring it when you first need it.

Reply:
Moved location declaration